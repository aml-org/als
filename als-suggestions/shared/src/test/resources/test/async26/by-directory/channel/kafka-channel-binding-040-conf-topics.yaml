asyncapi: 2.6.0

channels:
  some-channel:
    bindings:
      kafka:
        bindingVersion: 0.4.0
        topicConfiguration:
          *
