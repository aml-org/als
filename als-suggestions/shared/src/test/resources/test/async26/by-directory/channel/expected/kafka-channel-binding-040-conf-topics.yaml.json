[
  {
    "label": "cleanupPolicy",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "cleanupPolicy",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11cleanupPolicy",
    "filterText": "cleanupPolicy:\n  - ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 8,
          "character": 10
        },
        "end": {
          "line": 8,
          "character": 10
        }
      },
      "newText": "cleanupPolicy:\n  - "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "deleteRetentionMs",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "deleteRetentionMs",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11deleteRetentionMs",
    "filterText": "deleteRetentionMs: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 8,
          "character": 10
        },
        "end": {
          "line": 8,
          "character": 10
        }
      },
      "newText": "deleteRetentionMs: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "maxMessageBytes",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "maxMessageBytes",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11maxMessageBytes",
    "filterText": "maxMessageBytes: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 8,
          "character": 10
        },
        "end": {
          "line": 8,
          "character": 10
        }
      },
      "newText": "maxMessageBytes: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "retentionBytes",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "retentionBytes",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11retentionBytes",
    "filterText": "retentionBytes: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 8,
          "character": 10
        },
        "end": {
          "line": 8,
          "character": 10
        }
      },
      "newText": "retentionBytes: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "retentionMs",
    "kind": [
      
    ],
    "detail": "unknown",
    "documentation": "retentionMs",
    "deprecated": [
      
    ],
    "preselect": [
      
    ],
    "sortText": "11retentionMs",
    "filterText": "retentionMs: ",
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": 8,
          "character": 10
        },
        "end": {
          "line": 8,
          "character": 10
        }
      },
      "newText": "retentionMs: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]